setup/www/resources/config/nodejs.org

```diff
@@ -45,6 +45,8 @@ server {
     }
 
     location / {
+        try_files $uri $uri/ @english_fallback;
```

Comment:
Or even `try_files $uri $uri/ @english_fallback =404;`?  See the docs at http://nginx.org/en/docs/http/ngx_http_core_module.html#try_files